packages/nbmodel/src/ymodel.ts

```diff
@@ -63,6 +63,69 @@ export class YNotebook implements nbmodel.ISharedNotebook {
     this.ydoc.transact(f, undoable ? this : null);
   }
 
+  /**
+   * Gets notebook's source.
+   *
+   * @returns notebook's source.
+   */
+  getSource(): string {
```

Comment:
The `nbformat` doesn't specify a `source` for notebooks. Can you please refactor this without adding these methods? 


The intention of `nbmodel` is to properly reflect how a Jupyter notebook works. We want to properly integrate into the existing jupyterlab repository and we can reuse the features of `nbmodel` to model pure text documents as well. But we shouldn't inherit the duality of a `jupyterlab/notebook` to reflect both a notebook and a text-cell. I think these should be two different things.

Reply:
Already removed my last commit that had these changes.

Reply:
Resolving conversation as last commit removes it.